docs/_documentations/che-installinfo.md

```diff
@@ -64,60 +95,14 @@ Run the following command to install Che on OpenShift with `chectl`:
    $ chectl server:start --platform=k8s --installer=operator --domain=<ingress-domain> --che-operator-cr-yaml=codewind-checluster.yaml --che-operator-image=quay.io/eclipse/che-operator:7.9.2
    ```
 
-## Updating an existing Che installation
+#### Updating an existing Che installation to use Codewind
 
 If you already have a Che installation with TLS, you can update it for Codewind.
```

Comment:
@markcor11 These instructions are for 'installling' the Codewind extension onto a pre-existing Che installation. So this title needs to be rewritten to avoid the confusion with upgrading the Che version. I propose:

`Installing Codewind onto an existing Eclipse Che cluster`

Thoughts?

Reply:
yes that's fine,  just wasn't clear if it was upgrading che software,  or patching che just for codewind